app/components/primer/alpha/select_panel_element.ts

```diff
@@ -984,7 +985,7 @@ export class SelectPanelElement extends HTMLElement {
 
       for (const selectedItem of this.selectedItems) {
         const newInput = document.createElement('input')
-        newInput.setAttribute('data-list-input', 'true')
+        newInput.setAttribute('data-select-panel-input', 'true')
```

Comment:
Change the search term so we don't accidentally look for ActionList's form_builder input.